File: tests/fixtures/go_project/pricing/pricing.go
// Package pricing — fixture for tests.
package pricing

// CalculatePrice returns the final price for an item.
func CalculatePrice(itemID int, qty int) float64 {
	base := getBasePrice(itemID)
	if qty > 100 {
		base = applyDiscount(base, 10.0)
	}
	return base * float64(qty)
}

// applyDiscount applies a percentage discount (unexported).
func applyDiscount(price float64, pct float64) float64 {
	return price * (1.0 - pct/100.0)
}

func getBasePrice(itemID int) float64 {
	return float64(itemID) * 10.0
}

File: tests/fixtures/csharp_project/Pricing.cs
using System;

namespace Example
{
    /// <summary>Pricing module — fixture for tests.</summary>
    public class Pricing
    {
        /// <summary>Calculate final price including discount.</summary>
        public double CalculatePrice(int itemId, int qty)
        {
            double basePrice = GetBasePrice(itemId);
            if (qty > 100)
            {
                basePrice = ApplyDiscount(basePrice, 10.0);
            }
            return basePrice * qty;
        }

        private double ApplyDiscount(double price, double pct)
        {
            return price * (1.0 - pct / 100.0);
        }

        private double GetBasePrice(int itemId)
        {
            return itemId * 10.0;
        }
    }
}
